package.json

```diff
@@ -158,5 +158,14 @@
       "pre-commit": "npm run -s install-if-no-packages && node bin/pre-commit-hook.js",
       "pre-push": "npm run -s install-if-no-packages && node bin/pre-push-hook.js"
     }
-  }
+  },
+  "files": [
+    "dist/**/*.css",
+    "dist/**/*.js",
+    "dist/feature-config-core.php",
+    "includes/class-wc-admin-loader.php",
+    "includes/onboarding/*.php",
```

Comment:
Each added feature will also have to add a line here too, right? (just checking that I understand the process)

Reply:
Yes, unfortunately. I want to do some more investigation to see if there is a better way than having to add each one here. If we do end up having to add each one, I'll make sure that is documented better.

Reply:
I think a different folder + greedy search might be best, since it could be easy to accidentally forget to update this.

Reply:
👍 I've moved all of these to `includes/features/` so we can just look at `includes/features/**/*.php`.